chatbot-core/api/config/loader.py

```diff
@@ -14,8 +17,20 @@ def load_config():
     Returns:
         dict: Parsed configuration values.
     """
-    config_path = os.path.join(os.path.dirname(__file__), "config.yml")
+    file_dir = os.path.dirname(__file__)
+    config_dev_path = os.path.join(file_dir, "config.yml")
+    config_testing_path = os.path.join(file_dir, "config-testing.yml")
+
+    if os.path.isfile(config_testing_path):
```

Comment:
Check exists, so when we want to use the default one, dev may need to "delete" the test config.

Reply:
Would be great